.env.local

```diff
@@ -3,9 +3,11 @@ LOCK_TIMESERIES_PERIOD_FEATURE_FLAG: true
 COMPARISON_FEATURE_FLAG: true
 AAS_LIST_FEATURE_FLAG: true
 AAS_REPO_API_URL: 'http://localhost:5064/repo'
+SUBMODEL_REPO_API_URL: 'http://localhost:5064/repo'
```

Comment:
Do you plan to add this variable to the documentation in another PR?

Reply:
documentation is now updated